network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/quote.html

```diff
@@ -1,6 +1,6 @@
 {% load wagtailimages_tags %}
 
-<div class="row section-quote section-padding">
+<div class="row section-tw-quote section-padding">
```

Comment:
I think this one is off. The state list of classes only contains a `tw-quote` but not `section-tw-quote`. This might be a find and replace issue.